data/saas/dataset/mailchimp_transactional_dataset.yml

```diff
@@ -0,0 +1,5 @@
+dataset:
+  - fides_key: <instance_fides_key>
+    name: Mailchimp Transactional Dataset
+    description: A sample dataset representing the Mailchimp Transactional connector for Fides
+    collections: []
```

Comment:
Placeholder dataset for now, as we don't yet have a need for collections to be defined, although this is subject to change.